routes/qrcode.js

```diff
@@ -495,20 +506,40 @@ router.post(
         page_name: 'dashboard_qrcode',
       }); //should add error array here
     } else {
+      //check environment product was saved in for URL
+      let host = req.get('host');
+      let protocol = 'https';
+      // if running in dev then protocol can be http
+      if (process.env.NODE_ENV === CUSTOM_ENUMS.DEVELOPMENT) {
+        protocol = req.protocol;
+      }
+      let supplier_product = (
+        req.body.qrcode_company_name +
+        req.body.qrcode_product_name +
+        req.body.qrcode_contact_email
+      )
+        .split(' ')
+        .join('');
+      let hashID = hash.update(supplier_product).digest('base64');
```

Comment:
@DanielmAaron lets change this to digest('hex') i.e. `let hashID = hash.update(supplier_product).digest('hex');`  

So 'JuiceBros-GreenJuice-e=redacted@gmail.com' becomes 
'56e4c702f0def819d8f7a4d40567de3f477f843fc008add2d3462b465bcdef33'

instead of

'VuTHAvDe+BnY96TUBWfeP0d/hD/ACK3S00YrRlvN7zM='

Reply:
The hexadecimal in a URL probably looks better with only alphanumeric characters